src/libraries/System.Security.Cryptography.OpenSsl/src/Resources/Strings.resx

```diff
@@ -233,6 +233,9 @@
   <data name="Cryptography_NotValidPublicOrPrivateKey" xml:space="preserve">
     <value>Key is not a valid public or private key.</value>
   </data>
+  <data name="Cryptography_AlgorithmNotSupported" xml:space="preserve">
+    <value>Algorithm '{0}' is not supported on this platform.</value>
```

Comment:
On the one hand, the message is a bit weird in context.  On the other, it already exists, and it's certainly better than the app teardown that happens now.  So... seems like enough of an improvement for 6.0 to me.